packages/engine/lib/execution/src/worker/mod.rs

```diff
@@ -1,9 +1,5 @@
-//! The `worker` module defines the three different language runners for JavaScript, Python, and
-//! Rust and the accompanying API for their tasks and communication.
-//!
-//! The [`runner`] module contains the implementations for the languages and the
-//! [communication module](runner::comms). The [`task`] module defines tasks executed in the
-//! runners.
+//! Defines the three different language runners for JavaScript, Python, and Rust and the
```

Comment:
Maybe defines the ``[`Worker`]`` struct and the three language runners contained within it (maybe with doc links)

Reply:
The language runners are not defined here anymore. I changed the documentation to this:
```suggestion
//! Defines the task worker and accompanying API.
//!
//! The [`Worker`] contains and orchestrates the [language runners].
//! 
//! [language runners]: crate::runner
```